### PR TITLE
Updated ruamel.yaml import to avoid setup.py develop wheels issue

### PR DESCRIPTION
**Pull request by usmanazFIU**

### CODE REVIEW

**Comment by viniarck** [comment]
Hi @usmanazFIU, thanks for this fix and going through the tutorials. This fix looks safe to me, but let's also double check which `ruamel.yaml` version was installed in your venv to be sure, could please do these following things:

1) run `pip freeze | grep yaml`:

```
❯ pip freeze | grep yaml
ruamel.yaml==0.17.19
ruamel.yaml.clib==0.2.6
```

2) Could you also try to install this NApp again, without this PR, but instead of running `python3 setup.py develop` you run `pip install -e .`? For me it worked without this change here (although we should merge since the instructions/tutorials still use `python3 setup.py develop`, let's take this opportunity to confirm this on your end too.

**Comment by viniarck** [comment]
Make sure to also update the `CHANGELOG.RST` file accordingly mentioning what this change is doing, you can insert a line under the UNRELEASED Fixed section.

**Comment by usmanazFIU** [comment]
> pip freeze | grep yaml gave me same output. how ever pip install -e did not work out for me.  also I have changed CHANGELOG.RST and pushed the work again

File: CHANGELOG.rst [hunk 1]
```diff
@@ -20,6 +20,8 @@ Removed
 Fixed
 =====
 
+fixed napps.py bug - it used to give error before when python3 setup.py develop is executed.
```
Comment (viniarck) [inline CHANGELOG.rst:23]:
```suggestion
Updated ruamel.yaml import to avoid python3 setup.py develop wheels issue
```

**Comment by viniarck** [comment]
> > pip freeze | grep yaml gave me same output. how ever pip install -e did not work out for me.  also I have changed CHANGELOG.RST and pushed the work again

@usmanazFIU, thanks for pushing this commit. 

1 - Did you try to reinstall in the same venv?  It's worth removing it entirely. 

2 - Could you these commands below? Let's see if would work on your end too:

```
cd /tmp/
git clone https://github.com/kytos-ng/kytos-utils
cd kytos-utils
virtualenv -p python39 venv
source ./venv/bin/activate
pip install -e .
kytos napps list
```

3 - Let me know also which operating system you're using.

**Comment by usmanazFIU** [comment]
> ```
> kytos napps list
> ```

I have tried a new virtual env and the command now works and it installed successfully. should i now replace python3 setup.py develop with pip install -e.

**Comment by usmanazFIU** [comment]
2 - Could you these commands below? Let's see if would work on your end too:

Yes it worked on my end with new virtual environment

**Comment by viniarck** [comment]
@usmanazFIU, great. The commans will be replaced in a next opportunity on issue #28

**Review by viniarck — APPROVED** [review]
Appreciated your contribution @usmanazFIU

**Pull request merged**